### DISCUSSION TITLE
Oh no! There was a problem fetching the file from GitHub. The request responded with a 403 status. Please try again later.

### DISCUSSION THREAD

**Discussion by lili21** [remix-run/remix#8954] [Q&A]
`npx create-remix@latest --template remix-run/remix/templates/vite-express` doesn't work.

**Comment by kiliman** [comment]
You should try again. It's working for me.

<img width="869" alt="image" src="https://github.com/remix-run/remix/assets/47168/7196dffd-11ca-4231-8af8-74c46b9d2a5e">

**Reply by lili21** [reply]
hum. I'm in China, I can visit GitHub with a VPN. But I can't get the template in terminal. maybe I need set a proxy for terminal?

**Reply by iregina** [reply]
@lili21 I'm also getting the same error message (in Toronto) - were you able to create a new project?
